scripts/prepublish-stopper.js

```diff
@@ -1,4 +1,4 @@
-const ALLOWED = !!process.env.ALLOWED;
```

Comment:
@stevehobbsdev we originally added this flag check to prevent running directly `npm publish` from the root directory when a non-prod ready dist was available. With the use of our script, we need to tweak this. I'm basing this in the assumption that every CI provider exports `CI=true` for their builds. See [CircleCI variables](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables).

There's room for improvement here as I see we might not want to keep these scripts around anymore (now that we do it from the CI), but was thinking it might be wise to have still a manual fallback available to publish when the CD fails.

Reply:
As long as we can still do `npm publish` manually then we should be good, or have I misunderstood?

Reply:
At the end of the day I'm in favour of removing the scripts as long as the new bits prove to be reliable.

Reply:
Yes, I added the consideration of the `CI` env so we don't have to do further modifications and can still use it manually/locally if needed.